pkg/streamingpromql/testdata/ours/anchored.test

```diff
@@ -0,0 +1,458 @@
+load 15s
+  metric 1+1x4 9+1x4
```

Comment:
This abbreviated syntax is a bit confusing to me. Can we use the more explicit form?